Sources/StrongContractResponse/All/DataProtocol.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  DataProtocol.swift
+//
+//
+//  Created by Leonardo Dabus on 10/05/24.
+//
+
+import Foundation
+
+#if canImport(Crypto)
+import Crypto
+#endif
```

Comment:
Is one of these sufficient?  Why aren't we using elseif?

Reply:
IMO we should have different packages for each platform. This would avoid all trouble when running the server side.